exchanges/bitfinex/bitfinex_test.go

```diff
@@ -1331,8 +1331,8 @@ func TestWSSubscribedResponse(t *testing.T) {
 	assert.NoError(t, err, "Setting a matcher should not error")
 	err = b.wsHandleData([]byte(`{"event":"subscribed","channel":"ticker","chanId":224555,"subId":"waiter1","symbol":"tBTCUSD","pair":"BTCUSD"}`))
 	if assert.Error(t, err, "Should error if sub is not registered yet") {
-		assert.ErrorIs(t, err, stream.ErrSubscriptionFailure, "Should error SubFailure if sub isn't registered yet")
-		assert.ErrorIs(t, err, stream.ErrSubscriptionFailure, "Should error SubNotFound if sub isn't registered yet")
+		assert.ErrorIs(t, err, websocket.ErrSubscriptionFailure, "Should error SubFailure if sub isn't registered yet")
+		assert.ErrorIs(t, err, websocket.ErrSubscriptionFailure, "Should error SubNotFound if sub isn't registered yet")
```

Comment:
OOS: Need to change the error on this to subnotfound. As the check seems to be duplication.

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/b316ce2e2d